sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_custom_form_client.py

```diff
@@ -0,0 +1,374 @@
+                ),
+                use_label_file=True
+            ),
+            cls=lambda pipeline_response, _, response_headers: pipeline_response,
+            **kwargs
+        )
+
+        def callback(raw_response):
+            model = self._client._deserialize(Model, raw_response)
+            return CustomLabeledModel._from_generated(model)
+
+        deserialization_callback = cls if cls else callback
+        return LROPoller(self._client._client, response, deserialization_callback, LROBasePolling(**kwargs))
+
+    @distributed_trace
+    def begin_extract_form_pages(self, stream, model_id, **kwargs):
+        # type: (IO[bytes], str, Any) -> LROPoller
+        """Analyze Form.
+
+        :param stream: .pdf, .jpg, .png or .tiff type file stream.
+        :type stream: stream
+        :param str model_id: Model identifier.
+        :keyword bool include_text_details: Include text lines and element references in the result.
+        :keyword str content_type: Media type of the body sent to the API.
+        :return: LROPoller
+        :rtype: ~azure.core.polling.LROPoller
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        if isinstance(stream, six.string_types):
+            raise TypeError("Call begin_extract_form_pages_from_url() to analyze a document from a url.")
+
+        include_text_details = kwargs.pop("include_text_details", False)
+        content_type = kwargs.pop("content_type", None)
+        if content_type is None:
+            content_type = get_content_type(stream)
+
+        def callback(raw_response, _, headers):  # pylint: disable=unused-argument
+            extracted_form = self._client._deserialize(AnalyzeOperationResult, raw_response)
+            if extracted_form.analyze_result.document_results:
+                raise ValueError("Cannot call begin_extract_forms() with the ID of a model trained with labels. "
+                                 "Please call begin_extract_labeled_forms() instead.")
+            form_result = prepare_unlabeled_result(extracted_form, include_text_details)
+            return form_result
+
+        return self._client.begin_analyze_with_custom_model(
+            file_stream=stream,
+            model_id=model_id,
+            include_text_details=include_text_details,
+            content_type=content_type,
+            cls=kwargs.pop("cls", callback),
+            polling=LROBasePolling(timeout=POLLING_INTERVAL, **kwargs),
+            **kwargs
+        )
+
+    @distributed_trace
+    def begin_extract_form_pages_from_url(self, url, model_id, **kwargs):
+        # type: (str, str, Any) -> LROPoller
+        """Analyze Form.
+
+        :param url: The url of the document.
+        :type url: str
+        :param str model_id: Model identifier.
+        :keyword bool include_text_details: Include text lines and element references in the result.
+        :return: LROPoller
+        :rtype: ~azure.core.polling.LROPoller
```

Comment:
nitpick: can you include the return type of the LROPoller like this: https://github.com/Azure/azure-sdk-for-python/blob/4c40d733388a84121add16082ff966ab074ae99d/sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py#L78

Feel free to disregard this for a later pr

Reply:
Cool! I wasn't sure how to document this, but I will update